engine/table/src/main/java/io/deephaven/engine/util/AbstractScriptSession.java

```diff
@@ -325,38 +326,25 @@ public boolean hasParamName(String name) {
         @Override
         public <T> QueryScopeParam<T> createParam(final String name)
                 throws QueryScope.MissingVariableException {
-            if (!NameValidator.isValidQueryParameterName(name)) {
-                throw new QueryScope.MissingVariableException("Name " + name + " is invalid");
-            }
             return new QueryScopeParam<>(name, readParamValue(name));
         }
 
         @Override
         public <T> T readParamValue(final String name) throws QueryScope.MissingVariableException {
-            if (!NameValidator.isValidQueryParameterName(name)) {
-                throw new QueryScope.MissingVariableException("Name " + name + " is invalid");
-            }
-            // noinspection unchecked
-            return (T) getVariable(name);
+            return getVariable(name);
         }
 
         @Override
         public <T> T readParamValue(final String name, final T defaultValue) {
-            if (!NameValidator.isValidQueryParameterName(name)) {
-                return defaultValue;
-            }
-
             try {
-                // noinspection unchecked
-                return (T) getVariable(name);
+                return getVariable(name);
             } catch (MissingVariableException e) {
                 return defaultValue;
             }
         }
 
         @Override
         public <T> void putParam(final String name, final T value) {
-            NameValidator.validateQueryParameterName(name);
```

Comment:
Do you need the same change in standalone? Maybe we shouldn't change that, after all, but this does let us be more consistent with script sessions.